utils/docker-compose/scripts/generate.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+#
+# Copyright IBM Corp All Rights Reserved
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+export SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+export PATH=${SCRIPT_DIR}/../bin:${PWD}:$PATH
+export FABRIC_CFG_PATH=${SCRIPT_DIR}/../network-config
+CHANNEL_NAME=mychannel
+
+# remove previous crypto material and config transactions
+rm -fr ${FABRIC_CFG_PATH}/config/*
```

Comment:
you also should add a ``mkdir -p ${FABRIC_FG_PATH/config}'', i had to manually create this config directory or the script failed

Reply:
fixed